Snail/Observable.swift

```diff
@@ -4,7 +4,8 @@ public class Observable<T> : ObservableType {
     public typealias E = T
     private var isStopped: Int32 = 0
     var eventHandlers: [(Event<E>) -> Void] = []
-    private var test: [(next: ((T) -> Void)?, done: (() -> Void)?, error: ((Error) -> Void)?)] = []
+
+    public init() {}
```

Comment:
Don't we already do that: https://github.com/UrbanCompass/snail/blob/master/SnailTests/ObservableTests.swift#L19

it just uses the default constructor